package.json

```diff
@@ -28,8 +28,8 @@
     "lint": "yarn lint:sol && yarn lint:ts",
     "lint:sol": "solhint 'contracts/**/*.sol'",
     "lint:ts": "eslint 'test/**/*.ts' --max-warnings 0 --fix",
-    "typechain": "TS_NODE_TRANSPILE_ONLY=true yarn hardhat typechain",
```

Comment:
why was this changed?

Reply:
restored change to previous content i.e. `TS_NODE_TRANSPILE_ONLY=true hardhat typechain`. `yarn` was added as a fix in alpha.2 but it didn't work